---
geometry_msgs/Vector3 linear_acceleration
geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 linear_acceleration_filtered
geometry_msgs/Vector3 angular_velocity_filtered
